app/assets/javascripts/uploadcare/widget/tabs/camera-tab.coffee

```diff
@@ -11,10 +11,25 @@ uploadcare.namespace 'widget.tabs', (ns) ->
   class ns.CameraTab
 
     constructor: (@container, @tabButton, @dialogApi, @settings, @name) ->
-      if not @__checkCompatibility()
-        @dialogApi.hideTab(@name)
-        return
+      if @__checkCapture()
+        @container.append(tpl('tab-camera-capture'))
+        @container.addClass('uploadcare--camera')
+        handleFiles = (input) =>
+          @dialogApi.addFiles('object', input.files)
+          @dialogApi.switchTab('preview')
+        fileButton = @container.find('.uploadcare--camera__button_type_photo')
+        fileButton.on 'click', =>
+          utils.fileSelectDialog @container, {inputAcceptTypes: 'image/*'}, handleFiles, {capture: 'camera'}
+        fileButton = @container.find('.uploadcare--camera__button_type_video')
```

Comment:
I imagesOnly, the button (or a block with the button) should be hidden.

Reply:
Right! Thank you!